### PR TITLE
Peer Code Review ✅ 

### PR DESCRIPTION
**Pull request by victoria-reis**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,471 @@
...
+}
+
+figcaption,
+figure,
+main {
+  display: block;
+}
+
+figure {
+  margin: 1em 40px;
```
Comment (EstherEdell) [inline src/App.css:33]: fix indenting

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,471 @@
...
+}
+.form textarea:hover, .form textareafocus {
+  outline: 1px solid #93b9c4;
+}
+.form button {
+  border: none;
+  margin: 20px auto;
+  padding: 10px 20px;
+  border-radius: 20px;
+  background-color: #b4c2c3;
```
Comment (EstherEdell) [inline src/App.css:339]: extra semi-colon

**Review by EstherEdell — APPROVED** [review]
great job! left a few notes of things to look at!

File: src/App.js [hunk 3]
```diff
@@ -1,11 +1,82 @@
...
+import app from "./firebase";
+import { getDatabase, ref, onValue, remove } from "firebase/database";
+// components
+import Form from "./Components/Form";
+// styles
 import "./App.scss";
 
-function App() {
+const App = () => {
+	const [message, setMessage] = useState([]);
```
Comment (aalo826) [inline src/App.js:20]: Could put this in a separate component to keep App.js more organized!

File: src/App.js [hunk 4]
```diff
@@ -1,11 +1,82 @@
...
 	return (
-		<div className="App">
-			<h1>Self-love Reminder</h1>
-		</div>
+		<>
+			<main className="App wrapper">
+				<h1>Self-love Reminder</h1>
+				<h2>Your daily reminder to give yourself some love 🤍</h2>
+
+				<Form />
```
Comment (aalo826) [inline src/App.js:52]: good component usage!

File: src/Components/Form.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+import { getDatabase, ref, push } from "firebase/database";
+
+const Form = () => {
+	// firebase variables
+	const database = getDatabase(app);
+	const dbRef = ref(database);
+	// user's input state
+	const [userInput, setUserInput] = useState("");
+	// date variables
+	const today = new Date();
```
Comment (aalo826) [inline src/Components/Form.js:12]: Good use of comments to organize variables. 🙌

File: src/App.css [hunk 6]
```diff
@@ -0,0 +1,471 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400&family=Ms+Madi&display=swap");
```
Comment (aalo826) [inline src/App.css:1]: Could implement sass in the future to keep css more organized.

Comment (aalo826) [reply src/App.css:1]: Oh you are using sass actually, you could implement the use of a partial folder and organizing your css   
( _header.scss) ( _footer.scss) into the folder! Then you could @import those scss files in your partial folder into your App.scss.

File: src/Components/Form.js [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+	};
+
+	const handleUserSubmit = (event) => {
+		event.preventDefault();
+
+		if (userInput.trim()) {
+			push(dbRef, { text: userInput, date: messageDate });
+			setUserInput("");
+		} else {
+			alert("Please add a message!");
```
Comment (aalo826) [inline src/Components/Form.js:31]: Good alert message if user doesnt type anything! 

An alternative to the alert() method could be making a modal pop up! Benefits with this are that you can style the modal and change its size and what not. 

I remember for assignment 2 we had to integrate javascript in our project 1, I created a modal which was very simple. You can Google "how to make a modal" and there are a lot of good results that show you how to do it and how to style them.

File: src/firebase.js [hunk 8]
```diff
@@ -15,3 +15,5 @@ const firebaseConfig = {
 
 // Initialize Firebase
 const app = initializeApp(firebaseConfig);
+
```
Comment (aalo826) [inline src/firebase.js:18]: Victoria your website is so cool! Cool to see you used firebase.

**Pull request merged**